File: 351002/Ilian_Bukhovets/rest/internal/model/author.go
package model

type Author struct {
	ID        int64  `db:"id" json:"id"`
	Login     string `db:"login" json:"login"`
	Password  string `db:"password" json:"password"`
	FirstName string `db:"firstname" json:"firstname"`
	LastName  string `db:"lastname" json:"lastname"`
}

File: 351003/Komasilo/task310/src/main/java/by/bsuir/task310/model/Author.java
package by.bsuir.task310.model;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "tbl_author")
@Data
public class Author {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false, length = 64, unique = true)
    private String login;

    @Column(nullable = false, length = 128)
    private String password;

    @Column(nullable = false, length = 64)
    private String firstname;

    @Column(nullable = false, length = 64)
    private String lastname;

    @Column(nullable = false)
    private String role = "CUSTOMER";
}